### ISSUE TITLE
Material concept

### ISSUE THREAD

**Issue by Cybermaxke** [LanternPowered/LanternServer#19]
We need a material based concept, like in vanilla mc:
http://minecraft.gamepedia.com/Materials
These groups are used in multiple cases, one of these is defining which instrument will be used by a note block.

A thing to considers is to restructure the block properties so that these define the material of a block. Or a independent system, which seems silly if you can epand an existing system. I will need to think about about this before I start changing things.

**Comment by Meronat** [comment]
I agree that it would be best to reuse the block properties as most of the features of materials are already accounted for. 

Couldn't we just have a new InstrumentTypeProperty for blocks?

**Comment by Cybermaxke** [comment]
@Meronat That's why I added the property, but their are currently no groups for different base materials like wood or stone.
